routes/api/report/conversion.js

```diff
@@ -0,0 +1,267 @@
+const Joi = require('joi');
+const async = require('async');
+
+exports.reportcsv = {
+  path: '/api/report/conversion.csv',
```

Comment:
I like doing it this way, pulls the csv logic out of the main route, we should do that in the other places too. I'll create tasks